docs/content.zh/docs/connectors/datastream/dynamodb.md

```diff
@@ -0,0 +1,170 @@
+        sinkProperties.put(AWSConfigConstants.AWS_SECRET_ACCESS_KEY, "aws_secret_access_key");
+
+        ElementConverter<InputType, DynamoDbWriteRequest> elementConverter = new CustomElementConverter();
+
+        DynamoDbSink<String> dynamoDbSink =
+        DynamoDbSink.<InputType>builder()
+        .setDynamoDbProperties(sinkProperties)              // Required
+        .setTableName("my-dynamodb-table")                  // Required
+        .setElementConverter(elementConverter)              // Required
+        .setOverwriteByPartitionKeys(singletonList("key"))  // Optional  
+        .setFailOnError(false)                              // Optional
+        .setMaxBatchSize(25)                                // Optional
+        .setMaxInFlightRequests(50)                         // Optional
+        .setMaxBufferedRequests(10_000)                     // Optional
+        .setMaxTimeInBufferMS(5000)                         // Optional
+        .build();
+
+        flinkStream.sinkTo(dynamoDbSink);
+```
+{{< /tab >}}
+{{< tab "Scala" >}}
+```scala
+Properties sinkProperties = new Properties()
+// Required
+sinkProperties.put(AWSConfigConstants.AWS_REGION, "eu-west-1")
+// Optional, provide via alternative routes e.g. environment variables
+sinkProperties.put(AWSConfigConstants.AWS_ACCESS_KEY_ID, "aws_access_key_id")
+sinkProperties.put(AWSConfigConstants.AWS_SECRET_ACCESS_KEY, "aws_secret_access_key")
+
+val elementConverter = new CustomElementConverter();
+
+val dynamoDbSink =
+   DynamoDbSink.<InputType>builder()
+           .setDynamoDbProperties(sinkProperties)              // Required
+           .setTableName("my-dynamodb-table")                  // Required
+           .setElementConverter(elementConverter)              // Required
+           .setOverwriteByPartitionKeys(singletonList("key"))  // Optional    
+           .setFailOnError(false)                              // Optional
+           .setMaxBatchSize(25)                                // Optional
+           .setMaxInFlightRequests(50)                         // Optional
+           .setMaxBufferedRequests(10_000)                     // Optional
+           .setMaxTimeInBufferMS(5000)                         // Optional
+           .build()
+
+flinkStream.sinkTo(dynamoDbSink)
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+## Configurations
+
+Flink's DynamoDB sink is created by using the static builder `DynamoDBSink.<InputType>builder()`.
+
+1. __setDynamoDbProperties(Properties sinkProperties)__
+   * Required.
+   * Supplies credentials, region and other parameters to the DynamoDB client.
+2. __setTableName(String tableName)__
+   * Required.
+   * Name of the table to sink to.
+3. __setElementConverter(ElementConverter<InputType, DynamoDbWriteRequest> elementConverter)__
+   * Required.
+   * Converts generic records of type `InputType` to `DynamoDbWriteRequest`.
+4. __setOverwriteByPartitionKeys(List<String> partitionKeys)__
```

Comment:
Is this supposed to be formatted as italics because `Optional` and with default value?

Reply:
Thanks